src/Support/SupportRESTController.php

```diff
@@ -67,6 +67,19 @@ public static function get_registration_action() {
 	 */
 	public function register() {
 
+		register_rest_route(
+			$this->namespace,
+			'/get-diagnostic',
+			[
+				[
+					'methods'             => WP_REST_Server::READABLE,
+					'args'                => [],
+					'permission_callback' => [ $this, 'permission_callback' ],
+					'callback'            => [ $this, 'get_item' ],
+				],
+			]
+		);
+
```

Comment:
This endpoint isn't being used anywhere, and it's not validating the `args`. So I think it should be removed.